src/firebolt/model/V2/engine.py

```diff
@@ -26,15 +27,16 @@ class Engine(FireboltBaseModel):
 
     START_SQL: ClassVar[str] = 'START ENGINE "{}"'
     STOP_SQL: ClassVar[str] = 'STOP ENGINE "{}"'
-    ALTER_PREFIX_SQL: ClassVar[str] = 'ALTER ENGINE "{}" SET '
+    ALTER_PREFIX_SQL: ClassVar[str] = 'ALTER ENGINE "{}" '
     ALTER_PARAMETER_NAMES: ClassVar[Tuple] = (
         "NODES",
         "TYPE",
         "AUTO_STOP",
-        "RENAME_TO",
     )
     DROP_SQL: ClassVar[str] = 'DROP ENGINE "{}"'
 
+    _engine_name_re = re.compile(r"^[a-zA-Z0-9_]+$")
```

Comment:
If we keep the regex approach let's pop a comment above it - "engine names can only be alphanumeric with underscores". Saves a second having to read the regex itself.